source/opt/ir_loader.cpp

```diff
@@ -20,22 +20,22 @@
 namespace spvtools {
 namespace ir {
 
-IrLoader::IrLoader(const MessageConsumer& consumer, Module* module)
+IrLoader::IrLoader(const MessageConsumer& consumer, Module* m)
```

Comment:
Are you doing this to avoid shadowing?  Probably, and that's ok. :-)

Reply:
Yes this is because of shadowing.